catalog/core/catalog-core-commons/src/main/java/ddf/catalog/filter/delegate/CopyFilterDelegate.java

```diff
@@ -100,6 +100,12 @@ public Filter propertyIsEqualTo(String propertyName, String literal, boolean isC
     }
 
     @Override
+    public Filter propertyIsEqualTo(String functionName, List<Object> arguments, Object literal) {
+        throw new UnsupportedOperationException(
+                "propertyIsEqualTo(String,List,Object) not supported by CopyFilterDelegate.");
```

Comment:
Currently I am only passing the function name down so we can't recreate the filter exactly since it has no idea which parameters are literals and which are propertyNames. Conceivably it doesn't matter we could pass them around as literals since they get turned into strings anyway.